README.md

```diff
@@ -30,9 +30,9 @@ import "github.com/memcachier/mc/v3"
 // import "github.com/memcachier/mc"
 
 func main() {
-	// Error handling omitted for demo
+  // Error handling omitted for demo
 
-	// Only PLAIN SASL auth supported right now
+  // Only PLAIN SASL auth supported right now
```

Comment:
Please fix -- did you swap tabs for space? or vice-versa?

Reply:
Yes I did, before was very bad
Check it out how it is now https://github.com/patrickn2/mc/tree/chore--Update-Readme.md

Reply:
Just saw that is istill bad, Gonna fix asap

Reply:
Fixed